### PR TITLE
Prl 2179

### PR DESCRIPTION
**Pull request by praveenrach**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/PRL-2179

### Change description ###
save MIAM details to ccd
retrieve MIAM details from ccd and show the same in FE Citizen Journey - MIAM page

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/steps/tasklistresponse/miam/MIAMMapper.ts [hunk 1]
```diff
@@ -0,0 +1,43 @@
+import { CaseWithId } from '../../../app/case/case';
+import { Miam, Respondent, YesOrNo } from '../../../app/case/definition';
+import type { AppRequest } from '../../../app/controller/AppRequest';
+
+export const setMIAMDetails = (respondent: Respondent, req: AppRequest): Respondent => {
+  let miamFromResponsent: Miam;
+  if (respondent?.value?.response && respondent?.value?.response?.miam) {
+    miamFromResponsent = respondent?.value?.response?.miam;
```
Comment (cshiwarale) [inline src/main/steps/tasklistresponse/miam/MIAMMapper.ts:8]: spelling mistake - miamFromResponsent

File: src/main/steps/tasklistresponse/miam/MIAMPostController.ts [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+import { setMIAMDetails } from './MIAMMapper';
+
+@autobind
+export class MIAMPostController extends PostController<AnyObject> {
+  constructor(protected readonly fields: FormFields | FormFieldsFn) {
+    super(fields);
+  }
+  public async post(req: AppRequest, res: Response): Promise<void> {
+    const caseworkerUser = req.session.user;
```
Comment (cshiwarale) [inline src/main/steps/tasklistresponse/miam/MIAMPostController.ts:21]: should the variable name be 'loggedInCitizen'?

File: src/main/steps/tasklistresponse/miam/MIAMMapper.ts (hunk 1 above)
Comment (praveenrach) [reply src/main/steps/tasklistresponse/miam/MIAMMapper.ts:8]: updated the name as "miamFromRespondent"

File: src/main/steps/respondent/respondentcaseSequence.ts [hunk 3]
```diff
@@ -65,6 +74,16 @@ export const respondentCaseSequence: Step[] = [
     showInSection: Sections.AboutRespondentCase,
     getNextStep: () => RESPONDENT_TASK_LIST_URL,
   },
+  // {
+  //   url: CONSENT_TO_APPLICATION,
+  //   showInSection: Sections.AboutRespondentCase,
+  //   getNextStep: () => CONSENT_SUMMARY,
```
[the remark this answers is not in the document]
Comment (praveenrach) [reply src/main/steps/respondent/respondentcaseSequence.ts:80]: removed commented code

File: src/main/routes.ts [hunk 4]
```diff
@@ -5,11 +5,10 @@ import multer from 'multer';
 
 import { RespondentTaskListGetController } from '../main/steps/respondent/task-list/get';
 
-import { ConsentGetController } from './app/controller/ConsentGetController';
+//import { ConsentGetController } from './app/controller/ConsentGetController';
```
[the remark this answers is not in the document]
Comment (praveenrach) [reply src/main/routes.ts:8]: will remove the comment out code

File: src/main/app/document/DocumentManagementController.ts [hunk 5]
```diff
@@ -132,12 +132,12 @@ export class DocumentManagerController extends PostController<AnyObject> {
         endPoint = itemlist[itemlist.length - 2];
       }
 
-      const caseworkerUser = await getSystemUser();
-      req.session.user = caseworkerUser;
+      const loggedInCitizen = await getSystemUser();
+      //req.session.user = loggedInCitizen;
```
[the remark this answers is not in the document]
Comment (praveenrach) [reply src/main/app/document/DocumentManagementController.ts:136]: ConsentGetController.ts this is used for consent purpose, so we can ignore this file

File: src/main/app/controller/ConsentGetController.ts [hunk 6]
```diff
@@ -1,38 +0,0 @@
-import { Response } from 'express';
```
[the remark this answers is not in the document]
Comment (praveenrach) [reply src/main/app/controller/ConsentGetController.ts:1]: ConsentGetController.ts this is used for consent purpose, so we can ignore this file

File: src/main/routes.ts [hunk 7]
```diff
@@ -32,14 +33,16 @@ import {
   APPLICANT_TASK_LIST_URL,
   APPLICATION_MADE_IN_THESE_PRCEEDINGS,
   CITIZEN_DOWNLOAD_UPLOADED_DOCS,
-  CONSENT_SAVE,
-  CONSENT_TO_APPLICATION,
+  // CONSENT_SAVE,
```
[the remark this answers is not in the document]
Comment (praveenrach) [reply src/main/routes.ts:36]: ok, will remove the commented code

File: src/main/steps/tasklistresponse/miam/MIAMPostController.ts (hunk 2 above)
Comment (praveenrach) [reply src/main/steps/tasklistresponse/miam/MIAMPostController.ts:20]: done

**Pull request closed without merging**